plugins/teiid-8.11/src/main/java/org/komodo/teiid/TeiidInstanceImpl.java

```diff
@@ -253,6 +254,16 @@ public TeiidTranslator getTranslator(String name) throws Exception {
         return teiidTranslators;
     }
 
+    private boolean isDynamic(VDB vdb) {
```

Comment:
I am afraid not. The method uses both VDB and VDBMetaData which are Teiid classes. Since teiid-8.11 and teiid-8.12 depend on different versions of the Teiid libs then VDB, for example, is a different class type for each. Its impossible for both to use a common method since 8.11 VDB and 8.12 VDB are actually totally different classes.